src/providers/auth-factory.js

```diff
@@ -0,0 +1,35 @@
+(function (angular) {
+    'use strict';
+
+    var module = angular.module('angular-token-auth.auth', [
```

Comment:
I feel like this module name is too generic, but not sure what else to call it? `angular-token-auth.auth-factory`?